sklearn/gaussian_process/tests/test_kernels.py

```diff
@@ -51,8 +52,10 @@
 
 @pytest.mark.parametrize('kernel', kernels)
 def test_kernel_gradient(kernel):
+    def create_kernel(kernel, X):
+        return kernel(X, eval_gradient=True)
```

Comment:
You can ultimately delete this.
```suggestion
```

Reply:
@jjerphan 

Sorry, I missed this. I applied your suggestion.